db/migrate/20220916091845_create_sub_categories.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSubCategories < ActiveRecord::Migration[6.1]
+  def change
+    create_table :sub_categories do |t|
+      t.string :title
+      t.integer :categories_id
```

Comment:
rename column -> categories_id -> category_id
there should be an index of categories_id

Reply:
changed categories to category